React/Fabric/Mounting/ComponentViews/TextInput/RCTTextInputUtils.mm

```diff
@@ -38,6 +38,8 @@ void RCTCopyBackedTextInput(
   toTextInput.keyboardAppearance = fromTextInput.keyboardAppearance;
   toTextInput.spellCheckingType = fromTextInput.spellCheckingType;
   toTextInput.caretHidden = fromTextInput.caretHidden;
+  toTextInput.caretYOffset = fromTextInput.caretYOffset;
+  toTextInput.caretHeight = toTextInput.caretHeight;
```

Comment:
shouldn't be `toTextInput.caretHeight = fromTextInput.caretHeight` here?

Reply:
good catch! Fixed :-)